CONTRIBUTING.md

```diff
@@ -0,0 +1,80 @@
+# Development
+
+## Requirements
+
+* git
+* IntelliJ 2019.3+ Community Edition/Ultimate
+
+  * Install [Lombok plugin](https://plugins.jetbrains.com/plugin/6317-lombok/)
+  * Enable annotation processing (for Lombok): `File` -> `Settings` -> `Build`, `Execution`, `Deployment` -> `Compiler` -> `Annotation Processors` -> `Enable Annotation Processing`
+  * Set Project SDK to JDK 11: `Project Structure` -> `Project`
+
+## Set up project
+
+From the main project folder, run the following commands:
+
+```
+git config --local include.path ../.gitconfig
+ln -s ../../scripts/git-hooks/machete-pre-rebase .git/hooks/machete-pre-rebase
+ln -s ../../scripts/git-hooks/machete-status-branch .git/hooks/machete-status-branch
+ln -s ../../scripts/git-hooks/post-commit .git/hooks/post-commit
+ln -s ../../scripts/run-pre-build-checks .git/hooks/pre-commit
+```
+---
+Consider increasing maximum heap size for the IDE (the default value is 2048 MB). Go to `Help` -> `Change Memory Settings` for this.
+
+## Run/Debug
+
+To run an instance of IDE with Git Machete IntelliJ Plugin execute `:runIde` Gradle task (`Gradle panel` -> `Tasks` -> `intellij` -> `runIde` or `./gradlew runIde`).
```

Comment:
It seems that `./gradlew runIde` fails. :confused: Does it happen only to me?

Reply:
#280